backstory/src/main/java/com/google/sps/servlets/StoryManager.java

```diff
@@ -0,0 +1,55 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+import java.io.IOException;
+
+/**
+ * Interface for story generation class.
+ */
+
+public interface StoryManager {
+  /**
+   * Returns generation prefix.
+   *
+   * @return String The generation prefix.
+   */
+  public String getPrefix();
+
+  /**
+   * Returns maximum length for generation.
+   *
+   * @return int The maximum length for text generation.
+   */
+  public int getMaxLength();
```

Comment:
Will these getters be used by classes other than tests? Usually we try to avoid exposing a class's internals just for testing, because this makes it harder to refactor them.

Reply:
If the object is to be passed through to potentially document the parameters in an output page, I feel it good to maintain these getters for the time being.

Reply:
Well potentially if we wanted to display the given parameters post-generation we could have all of them in one object to get from versus text from a generation method from the object and the parameters from somewhere else - same for if we expand to add a share feature. I can rid them if need be but I think as far as mvp pre-integration it would be nice to maintain them for now.

Reply:
Removed the getters as they became completely unused with the exceptions implementation.